content/applications/hr/payroll/payroll_overview.rst

```diff
@@ -0,0 +1,90 @@
+============================
+Getting started with Payroll
+============================
+
+:guilabel:`Payroll` allows for management of employees, creation of contracts, approval of
+timesheets, generation of payslips, and in-depth reporting. The four main components of :guilabel
+`Payroll` are :guilabel:`Employees`, :guilabel:`Work Entries`, :guilabel:`Payslips`, and :guilabel:
+`Reporting`. All the tools needed to handle payroll accurately, and provide a birds-eye view of both
+employees and payments, are handled within these four aspects.
+
+Employees
+=========
+
+The list of employees directly overlaps with the :guilabel:`Employees` application, and displays all
+the employees of the business, whether full-time, part-time, or contract. The main :guilabel:
+`Employees` dashboard displays a summary of each employee’s information in their individual kanban
+card. Employees can be sorted by company and/or department from the left side navigation.
+
+.. image:: payroll_overview/employee-overview.png
+   :align: center
+   :alt: Employees dashboard view.
+
+Contracts
+---------
+
+In order for an employee to be paid, an active contract is required. The :guilabel:`Contracts`
+section, housed under :guilabel:`Employees`, displays all the contracts in a grid view. The default
+view displays both running contracts and contracts that need action. Expired and canceled contracts
+are hidden from the default view.
+
+.. image:: payroll_overview/contracts-overview.png
+   :align: center
+   :alt: Contracts dashboard view showing running contracts and contracts with issues.
+
+Work Entries
+============
+
+The :guilabel:`Work Entries` dashboard provides a visual overview of all employee's individual time
+sheets, with each day split into morning and afternoon shifts. From this page, work entries can be
+regenerated if there is an update to schedules, and payslips can be generated.
```

Comment:
Since this was just the overview doc, there are other docs coming, such as an in-depth work entries, contracts, etc doc. I couldn't add a "see also" since it isn't uploaded to github yet. This was an issue for a lot of my overview docs- not being able to link to a more detailed doc because I am either still writing them, OR because I can't link to a non-exiting doc and the overviews must be published first. Since we have not left the Payroll app for this entire doc, I thought it was clear, but I did add how to get to this section.